vtex-apps/place-components
# Add device coordinates component

#### What problem is this solving?

This PR adds the DeviceCoordinates component, that should ask for permissions to get your location from the device you are using, and it should automatically fill data based on that location.

[Related clubhouse story](https://app.clubhouse.io/vtex/story/31851/criar-devicecoordinates)

#### How should this be manually tested?

[Workspace](https://devicecoordinates--checkoutio.myvtex.com/places)

This won't currently work since the `apiKey` in the code is not the real one, that shouldn't be disclosed because this code is open source. You can `vtex link` this locally (don't forget to also link `address-graphql` and `styleguide`) and place a valid `apiKey` to see how this works.

#### Checklist/Reminders


#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements
